### ISSUE TITLE
Evaluate `visualize` styling with new mermaid update

### ISSUE THREAD

**Issue by zoeyTM** [NomicFoundation/hardhat-ignition#636]
I recently opened an [issue](https://github.com/mermaid-js/mermaid/issues/4935) in the mermaidjs repo asking for a config option to add padding around subgraph titles. That issue has now been closed and the feature PR merged into their develop branch, so, pending their [next release](https://github.com/mermaid-js/mermaid/tags) (presumably v10.6.2), we should reevaluate our graph styles with the new config setting and see if there's an improvement.

**Comment by kanej** [comment]
They dropped support for the feature we where waiting on. So this is on ice for the moment.

**Comment by zoeyTM** [comment]
The feature we were waiting on doesn't seem to have been tested on complex graphs and was broken in ways that makes it unusable for us. This issue is closed now until [this PR](https://github.com/mermaid-js/mermaid/issues/4813) gets merged, then we can reevaluate.
